UnityProject/Assets/Scripts/UI/Systems/Jobs/CrewManifestManager.cs

```diff
@@ -99,7 +99,7 @@ public SecurityRecord GenerateSecurityRecord(PlayerScript script, JobType jobTyp
 		{
 			SecurityRecord record = new SecurityRecord();
 
-			record.EntryName = script.playerName;
+			record.EntryName = script.characterSettings.Name;
```

Comment:
didn't we already push a similar fix? `git merge` moment?

Reply:
no? idk  Though